src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatterTest.java

```diff
@@ -45,7 +45,7 @@ public void testHTML() {
         assertEquals("{\\\"{a}}", formatter.format("&auml;"));
         assertEquals("{\\\"{a}}", formatter.format("&#228;"));
         assertEquals("{\\\"{a}}", formatter.format("&#xe4;"));
-        assertEquals("{$\\Epsilon$}", formatter.format("&Epsilon;"));
+        assertEquals("{{$\\Epsilon$}}", formatter.format("&Epsilon;"));
```

Comment:
It is not actually capitalization. It is about replacing the HTML/Unicode symbol with the correct LaTeX command. Consider `\Leftrightarrow`. And see #405. Btw, the cleanup doesn't handle this case so it isn't a solution (try `\AA`). I cannot say anything about Biblatex, but for BibTeX it is an actual problem.

For the umlaut, see:
http://tex.stackexchange.com/questions/57743/how-to-write-%C3%A4-and-other-umlauts-and-accented-letters-in-bibliography
https://en.wikibooks.org/wiki/LaTeX/Special_Characters
and consider using another accent, like `\k`. so `{\ka}` based on your suggestion. I read a very good explanation about why `{\"{a}}` and the like is the correct way, although other cases work in specific situations, but I cannot find the link right now. I doubt Biblatex has solved the final case, unless it is at the end of a word (LaTeX will complain if nothing else. Again, imagine `{\ka}` (which you can write `{\k a}` to get around the first problem), inside a word, so `b\kab` or `b\k ab` or `b\k a b`. I'm quite sure none of these will give the correct result.